### ISSUE TITLE
FiscalYear.calendar_for/1 raises

### ISSUE THREAD

**Issue by DaTrader** [elixir-cldr/cldr_calendars#10]
Not sure if this is a bug, but anyway:

```
iex(32)> Cldr.Calendar.FiscalYear.calendar_for( :US)          
** (ArgumentError) Option :month is replaced with :month_of_year
    (ex_cldr_calendars 1.17.1) lib/cldr/calendar/config.ex:170: Cldr.Calendar.Config.invalidate_old_options!/1
    (ex_cldr_calendars 1.17.1) lib/cldr/calendar/config.ex:68: Cldr.Calendar.Config.extract_options/1
    (ex_cldr_calendars 1.17.1) lib/cldr/calendar.ex:619: Cldr.Calendar.create_calendar/3
```

**Comment by kipcole9** [comment]
Commit [926abd4](https://github.com/elixir-cldr/cldr_calendars/commit/926abd4f520596a15b4bde9d7ff7728c703314b8) the rather egregious and long-standing bug is fixed. Thanks for the report. Will publish a new version after resolving the other open issues.  Here's the changelog entry:

### Bug Fixes

* Fixes `Cldr.Calendar.FiscalYear.calendar_for/2`. Thanks for @DaTrader for the report. Closes #10.

**Comment by kipcole9** [comment]
I have published [ex_cldr_calendars version 1.17.2](https://hex.pm/packages/ex_cldr_calendars/1.17.2) that includes the fix for this issue.
